source/manual/redirect-routes.html.md.erb

```diff
@@ -7,17 +7,131 @@ parent: "/manual.html"
+for that.
+
+You'll need to get the content ID of the current thing that exists at the
+route you want to redirect from. This is generally just a case of visiting
+the route prefixed with `/api/content` to expose its Content Store
+information.
+
+Now, log into the app console:
+
+`$ gds govuk connect app-console -e production publishing_api/publishing-api`
+
+You may want to log into the Content Store app console in another shell:
+
+`$ gds govuk connect app-console -e production content_store/content-store`
+
+In the Content Store console, run
+`content_item = ContentItem.find_by(content_id: "CONTENT_ID")` to get a
+handle on the current state of the content item. If the content item
+already has a `document_type` of `redirect` and the `redirects` property
+has the `path` and `destination` properties you're aiming for, then you
+may just need to [nudge Content Store into talking to Router API][[register-redirect-from-content-store-section].
+
+In the Publishing API console, make a call to `Unpublish`:
+
+```ruby
+Commands::V2::Unpublish.call(
+  content_id: "CONTENT_ID",
+  type: "redirect",
+  explanation: "manually redirected by YOUR_NAME",
+  alternative_path: "/new-path",
+  discard_drafts: true,
+  unpublished_at: Time.now,
+)
+```
+
+You should now see an Unpublishing record in Publishing API:
+
+```ruby
+Unpublishing.last
+=> #<Unpublishing id: 435518, edition_id: 1, type: "redirect", explanation: "manually redirected by Chris", alternative_path: "/new-path", created_at: "2020-11-27 14:03:08", updated_at: "2020-11-27 14:20:47", unpublished_at: nil, redirects: [{"path"=>"/old-path", "type"=>"exact", "destination"=>"/new-path"}]>
+```
+
+And you should see that the content item in the Content Store
+console has been updated, if you run `content_item.reload`.
+
+[Content Store talks directly to Router API](https://github.com/alphagov/content-store/blob/dd79a03d74f130650bc97d1c84aae557ccea58d3/app/models/content_item.rb#L33)
+to update the route. Now when you visit the route in your
+browser, you should be redirected.
+
+### Manually register a route in Content Store
+[register-redirect-from-content-store-section]: #manually-register-a-route-in-content-store
+
+Sometimes Content Store is unsuccessful in updating a route in
+Router API, e.g. if the call from Content Store happens while
+the Router machines are being deployed.
+
+In such cases, you can get Content Store to register the route
+again, by opening the Content Store console and running:
+
+```ruby
+ContentItem.find_by(base_path: "/old-path-to-be-redirected").route_set.register!
+```
+
+### Manually apply a redirect in Router API
+
+You might need to urgently apply a redirect, and Publishing API
+might have a large backlog of publishing in its downstream_high
+queue, making the Publishing API approach too slow.
+
+First, connect to the Router API console:
+
+`$ gds govuk connect -e production app-console draft_cache/router-api`
```

Comment:
This will only update the route in the draft stack, which is fine for the docs, but might be easy for someone who wants to do this in the live stack to miss.

Reply:
Good spot. I actually can't seem to connect to the app console on the live stack 🤔 

```
$gds govuk connect app-console -e integration cache/router-api

/usr/local/bin/govuk_app_console: 17: cd: can't cd to /var/apps/router-api
```

Reply:
It's actually on a `router_backend` machine class! But I think you should be able to do `gds govuk connect app-console -e integration router-api` and it will find the appropriate backend for you.